plugins/contrast/index.js

```diff
@@ -38,17 +45,20 @@ class ContrastPlugin extends Plugin {
             suggestedColorsRatio: suggestedColors.contrast
         };
 
-        // Add click handler to preview checkbox.
+        // Add click handler to preview checkbox
         let $description = $(descriptionTemplate(templateData));
+        let originalFgColor = style.color;
+        let originalBgColor = style.backgroundColor;
+
         $description.find(".preview-contrast-fix").click((e) => {
             if ($(e.target).prop("checked")) {
-                // Set suggested colors.
+                // Set suggested colors
                 $(el).css("color", suggestedColors.fg);
                 $(el).css("background-color", suggestedColors.bg);
             } else {
-                // Set original colors.
-                $(el).css("color", axs.color.colorToString(fgColor));
-                $(el).css("background-color", axs.color.colorToString(bgColor));
+                // Set original colors
```

Comment:
It would be nice to DRY this up and combine it with whatever functionality is located on line 161.